docs/LOGGING.md

```diff
@@ -115,7 +116,7 @@ The uploading is controlled on a scheduled timer. When the timer fires, the "log
 
 The "loguploader" collects stats of round-trip delay, controller CPU load percentage and log batch processing time. The current EVE implementation does not use those stats in calculating the uploading timer values.
 
-The already uploaded gzip files with app logs are moved to /persist/newlog/keepSentQueue directory or removed in case of dev.log.upload files.
+The already uploaded gzip files with app logs are moved to /persist/newlog/keepSentQueue directory or removed in case of dev.log files.
```

Comment:
All 'dev.log*' named files can not be removed. can they?

Reply:
I think they have a copy in the 'keep' directory already in the latest newlog change. so they should be removed.

Reply:
@naiming-zededa is right, but I understand that the sentence is confusing - I'll rewrite